jupyter_collaboration/stores.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+from traitlets import Int, Unicode
+from traitlets.config import LoggingConfigurable
+from ypy_websocket.ystore import SQLiteYStore as _SQLiteYStore
+from ypy_websocket.ystore import TempFileYStore as _TempFileYStore
+
+
+class TempFileYStore(_TempFileYStore):
+    prefix_dir = "jupyter_ystore_"
+
+
+class SQLiteYStoreMetaclass(type(LoggingConfigurable), type(_SQLiteYStore)):  # type: ignore
```

Comment:
Could you add a comment why we need this complex code based on metaclass rather than the classical direct inheritance?

Reply:
I just moved it to a new file. I'm not sure why we need it. I have not looked into the stores yet.

@davidbrochart Could you open a PR adding documentation?